app/models/story.rb

```diff
@@ -1,4 +1,11 @@
 class Story < ActiveRecord::Base
+  module ReadOnlyDocuments
+    def documents=(attachments)
```

Comment:
nope, the "documents=" is defined dynamically by `has_attachments :documents` from attachinary. the AR `has_many` can have a `readonly: true` but this is defined in the class level and I didn't find any way to redefine readonly association on the fly. But again, documents is not an association it's a method that links to Cloudinary::Upload underneath. Didn't find any easy way to disallow the setter on a readonly model.